torchvision/models/quantization/inception.py

```diff
@@ -0,0 +1,254 @@
+        return self.myop.cat(outputs, 1)
+
+
+class QuantizableInceptionB(inception_module.InceptionB):
+    def __init__(self, *args, **kwargs):
+        super().__init__(BasicConv2d=QuantizableBasicConv2d, *args, **kwargs)
+        self.myop = nn.quantized.FloatFunctional()
+
+    def forward(self, x):
+        outputs = self._forward(x)
+        return self.myop.cat(outputs, 1)
+
+
+class QuantizableInceptionC(inception_module.InceptionC):
+    def __init__(self, *args, **kwargs):
+        super().__init__(BasicConv2d=QuantizableBasicConv2d, *args, **kwargs)
+        self.myop = nn.quantized.FloatFunctional()
+
+    def forward(self, x):
+        outputs = self._forward(x)
+        return self.myop.cat(outputs, 1)
+
+
+class QuantizableInceptionD(inception_module.InceptionD):
+    def __init__(self, *args, **kwargs):
+        super().__init__(BasicConv2d=QuantizableBasicConv2d, *args, **kwargs)
+        self.myop = nn.quantized.FloatFunctional()
+
+    def forward(self, x):
+        outputs = self._forward(x)
+        return self.myop.cat(outputs, 1)
+
+
+class QuantizableInceptionE(inception_module.InceptionE):
+    def __init__(self, *args, **kwargs):
+        super().__init__(BasicConv2d=QuantizableBasicConv2d, *args, **kwargs)
+        self.myop = nn.quantized.FloatFunctional()
+
+    def _forward(self, x):
+        branch1x1 = self.branch1x1(x)
+
+        branch3x3 = self.branch3x3_1(x)
+        branch3x3 = [self.branch3x3_2a(branch3x3), self.branch3x3_2b(branch3x3)]
+        branch3x3 = self.myop.cat(branch3x3, 1)
+
+        branch3x3dbl = self.branch3x3dbl_1(x)
+        branch3x3dbl = self.branch3x3dbl_2(branch3x3dbl)
+        branch3x3dbl = [
+            self.branch3x3dbl_3a(branch3x3dbl),
+            self.branch3x3dbl_3b(branch3x3dbl),
+        ]
+        branch3x3dbl = self.myop.cat(branch3x3dbl, 1)
+
+        branch_pool = F.avg_pool2d(x, kernel_size=3, stride=1, padding=1)
+        branch_pool = self.branch_pool(branch_pool)
+
+        outputs = [branch1x1, branch3x3, branch3x3dbl, branch_pool]
+        return outputs
+
+    def forward(self, x):
+        outputs = self._forward(x)
+        return self.myop.cat(outputs, 1)
+
+
+class QuantizableInception3(inception_module.Inception3):
+    def __init__(self, num_classes=1000, aux_logits=True, transform_input=False):
+        super().__init__(
+            num_classes=num_classes,
+            aux_logits=aux_logits,
+            transform_input=transform_input,
+            BasicConv2d=QuantizableBasicConv2d,
+            InceptionA=QuantizableInceptionA,
+            InceptionB=QuantizableInceptionB,
+            InceptionC=QuantizableInceptionC,
+            InceptionD=QuantizableInceptionD,
+            InceptionE=QuantizableInceptionE,
+        )
+        self.quant = torch.quantization.QuantStub()
+        self.dequant = torch.quantization.DeQuantStub()
+
+    def forward(self, x):
+        x = self._transform_input(x)
+        x = self.quant(x)
+        # N x 3 x 299 x 299
```

Comment:
Similar comments as for googlenet: Can this code be shared with the float model?

Reply:
The reason is the same as googlenet, it can't because of the jit code added to make the model scriptable.

Reply:
Ok, I have a suggestion above, if that works out then we can try the same here.